provision/acc_provision/templates/acc-provision-configmap.yaml

```diff
@@ -104,6 +104,24 @@ data:
                 {% endif %}
                 {% endfor %}
                 }
+                {% elif key == 'service_graph_endpoint_add_delay' %}
+                "service_graph_endpoint_add_delay": {
+                {% for delaykey, delayvalue in config.kube_config.service_graph_endpoint_add_delay.items() %}
+                    {% if delaykey == 'services' %}
+                    "services": [
+                        {% for delaysvc in config.kube_config.service_graph_endpoint_add_delay.services %}
+                        {
+                            {% for delaysvckey, delaysvcvalue in delaysvc.items() %}
+                            {{ delaysvckey|json }}: {{ delaysvcvalue|json|indent(width=20) }}{{ ", " if not loop.last else "" }}
+                            {% endfor %}
+                        }{{ ", " if not loop.last else "" }}
+                        {% endfor %}
+                    ]{{ ", " if not loop.last else "" }}
+                    {% else %}
+                    {{ delaykey|json }}: {{ delayvalue|json|indent(width=20) }}{{ ", " if not loop.last else "" }}
+                    {% endif %}
+                {% endfor %}
+                }{{ ", " if not loop.last else "" }}
```

Comment:
can we look for specific keys "name" and "namespace" (to avoid sending erroneous input)?